action.yml

```diff
@@ -20,33 +20,33 @@ inputs:
     default: ''
 
 runs:
-  using: composite
+  using: 'composite'
   steps:
-    - name: Add .ssh directory
+    - name: 'Add .ssh directory'
       run: |
         mkdir -p ~/.ssh
         chmod 700 ~/.ssh
       shell: bash
 
-    - name: Copy private key
+    - name: 'Copy private key'
+      shell: 'bash'
```

Comment:
`shell` comes before executing `run`

Reply:
> `shell` is read before executing `run`

I see no mention of that in [GitHub Docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsrun).

Reply:
This is simply a logical conclusion.
To keep things in chronological order.